From ea4ca50ca9cab755f3722096158a46deec0e663a Mon Sep 17 00:00:00 2001
From: Arthur She <arthur.she@linaro.org>
Date: Tue, 25 Aug 2020 09:51:26 -0700
Subject: [PATCH] src/Makefile: Fix Makefile macro error

This error caused the curl library wouldn't be linked correctly.

Upstream-Status: Pending
Signed-off-by: Arthur She <arthur.she@linaro.org>
---
 src/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index 1cdf8b0..acd04dd 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -57,7 +57,7 @@ ifeq ($(MUST_BUILD_CURL_CLIENT),yes)
   TRANSPORT_MODS += $(BLDDIR)/lib/curl_transport/xmlrpc_curl_transport
   TRANSPORT_MODS += $(BLDDIR)/lib/curl_transport/curltransaction
   TRANSPORT_MODS += $(BLDDIR)/lib/curl_transport/curlmulti
-  TRANSPORT_LIBDEP += $(shell $CURL_CONFIG --libs)
+  TRANSPORT_LIBDEP += $(shell $(CURL_CONFIG) --libs)
   TRANSPORT_INCLUDES += -Isrcdir/lib/curl_transport
 endif
 ifeq ($(MUST_BUILD_LIBWWW_CLIENT),yes)
-- 
2.17.1

